### DISCUSSION TITLE
help: i'm going crazy trying to debug this theme file!!

### DISCUSSION THREAD

**Discussion by rahil627** [helix-editor/helix#11776] [Troubleshooting] [answered]
i'm just trying to salvage [a helix port](https://github.com/daltonmenezes/aura-theme/pull/211) of the [aura theme](https://marketplace.visualstudio.com/items?itemName=DaltonMenezes.aura-theme) which seems abandoned.. _i'm guessing because adding a theme to the main repo was/is a pain in the butt (build node.js project with a million deps..)_

[this one has a bug in it](https://github.com/rahil627/my-programming-stuff/blob/main/tools/themes/aura-template-helix.toml)

[this one is the same thing, but i hard-replaced the colors](https://github.com/rahil627/my-programming-stuff/blob/main/tools/themes/aura-soft-dark-helix.toml), and it works!!

i wonder, is there a way to debug it, similar to how the config file shows errors..??

all it got right were the strings:
![image](https://github.com/user-attachments/assets/4c2ca49e-c35d-405a-8ed6-35467151588e)

**Comment by galearez** [comment, marked as answer]
There is a typo in line 79

**Reply by rahil627** [reply]
😭

**Reply by rahil627** [reply]
oh, my goodness... welp, i _just_ ended up re-writing it all... _uuuughuuuugh my engrish_.. :sob:

thank you though!! i'll see if there's any goodies to scrape from the old version..

just tree-sitter, no lsp:
![image](https://github.com/user-attachments/assets/544a51bc-bfaa-4205-acd6-90e6f6b13afe)

haven't decided if the blink effect upon selection is a great idea yet.. 😄 but it's surprisingly great in many places!

**Reply by kirawi** [reply]
For future reference, theme errors are logged and you can see them via `:log-open`

**Reply by rahil627** [reply]
ooooh snap, thanks so much! yeah, i think once the theme was messed up, i couldn't see the printed error.. good to know!
